### DISCUSSION TITLE
CVE-2024-37287 (CVSS 9.9): Kibana Patch for Severe Security Vulnerability

### DISCUSSION THREAD

**Discussion by aravindameya** [wazuh/wazuh#25102] [Q&A] [answered]
Hi Team,

Does Wazuh uses kibana or Opensearch dashboard?

https://securityonline.info/cve-2024-37287-cvss-9-9-urgent-kibana-patch-for-severe-security-vulnerability/
https://discuss.elastic.co/t/kibana-8-14-2-7-17-23-security-update-esa-2024-22/364424

Does the mentioned vulnerability applicable for Wazuh also? 
Please clear me on this.

Thanks in advance.

**Comment by matias-braida** [comment]
Hello aravindameya,

The Wazuh dashboard is based on "OpenSearch Dashboards" since Wazuh version 4.3.0.
Please take a look at this link: https://wazuh.com/blog/introducing-wazuh-4-3-0/

What Wazuh version do you have installed?

Regards.

**Reply by aravindameya** [reply]
Hi Matias,

I have been using Wazuh v4.7.4

Thanks for your response.

**Reply by matias-braida** [reply, marked as answer]
As you are running a version over v4.3.0 you shouldn't be affected by this CVE.

**Reply by aravindameya** [reply]
Thanks for the clarification.
